saleor/product/tests/test_category.py

```diff
@@ -55,14 +56,15 @@ def test_delete_categories_trigger_product_updated_webhook(
     any_webhook,
     settings,
 ):
-    # given
```

Comment:
Instead of removing `#given` you should add missing `when` and `then` 😄

Reply:
that one was removed by mistake :D - I've put it back with the other words missing